third_party/ros/roslaunch/BUILD.bazel

```diff
@@ -1,21 +1,36 @@
 load("@rules_python//python:defs.bzl", "py_binary", "py_library")
 load("@rules_ros_pip_deps//:requirements.bzl", "requirement")
 load("//build_tools:pylint.bzl", "pylint")
-load("//third_party:expand_template.bzl", "expand_template")
 
-expand_template(
+genrule(
     name = "deps",
-    out = "deps.py",
-    data = ["roscore.xml"],
-    substitutions = {"{roscore_xml_path}": "$(rootpath roscore.xml)"},
-    template = "deps.py.tpl",
+    outs = ["deps.py"],
+    cmd = """cat <<EOF > $@
+ROSMASTER_PATH = '$(rootpath @ros_comm//:rosmaster)'
+ROSCORE_XML_PATH = '$(rootpath roscore.xml)'
+EOF""",
+    srcs = ["roscore.xml"],
+    tools = ["@ros_comm//:rosmaster"],
```

Comment:
rosmaster should be IMO in srcs, if it's in tools then the target is going to be built in exec config.

Reply:
It'll be built in the `exec` config in order to get the correct path out, but is that a problem for a python script?

Reply:
In case this is a worry, I've updated it so that the tooling appears in `srcs`

Reply:
This is now what I get when I cherry-pick this onto my bzlmod branch:
```
$ cat bazel-bin/third_party/ros/roslaunch/roscore.xml 
<launch>
  <group ns="/">
    <node type="external/_main~non_module_dependencies~ros_comm/rosout" name="rosout" respawn="true"/>
  </group>
</launch>
$ cat bazel-bin/third_party/ros/roslaunch/deps.py
ROSMASTER_PATH = 'external/_main~non_module_dependencies~ros_comm/rosmaster'
ROSCORE_XML_PATH = 'third_party/ros/roslaunch/roscore.xml'
```